app/services/authentication/providers/facebook.rb

```diff
@@ -0,0 +1,57 @@
+module Authentication
+  module Providers
+    # Facbook authentication provider, uses omniauth-facebook as backend
+    class Facebook < Provider
+      OFFICIAL_NAME = "Facebook".freeze
+      SETTINGS_URL = "https://www.facebook.com/settings?tab=applications".freeze
+
+      def new_user_data
+        {
+          name: @info.name,
+          email: @info.email || "",
```

Comment:
should we add a test for the missing email?

Reply:
Yes, will do!